jjb/edgex-taf/edgex-taf-common.yaml

```diff
@@ -0,0 +1,16 @@
+---
+
+- project:
+    name: edgex-taf-common
+    project-name: edgex-taf-common
+    project: edgex-taf-common
+    mvn-settings: edgex-taf-common-settings
+    jenkins_file: 'Jenkinsfile'
```

Comment:
Technically this is not needed. This is the default value.

Reply:
Do you mean remove the `jenkins_file: 'Jenkinsfile'` ?